vending-app/build.gradle

```diff
@@ -17,6 +17,8 @@ android {
         versionCode vendingAppVersionCode
         minSdkVersion androidMinSdk
         targetSdkVersion androidTargetSdk
+
+        multiDexEnabled true
```

Comment:
It seems the pipeline is failing due to the 65,536 method limit? I'm quite confused, as this is not happening locally. Anyhow, this commit is supposed to enable multidex, but I'm unsure if that's an appropriate solution in this case.

Reply:
Now it have this lint error:
```
/home/runner/work/GmsCore/GmsCore/vending-app/src/main/java/com/android/vending/licensing/LicenseServiceNotificationRunnable.java:95: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          notificationManager.notify(callerUid, notification);
```

Did you run also lint locally?

Reply:
@ale5000-git I noticed the pipeline failing, but not, I haven't run the lint locally. I likely won't have time to fix this issue in the next days.

Reply:
No problem, take your time.
Your contribution is really appreciated.